### PR TITLE
full rework jwt work, add access\refresh token

### PR DESCRIPTION
**Pull request by DoctorFreed**

### CODE REVIEW

File: src/modules/controllers/authorization.controller.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const jwt = require("jsonwebtoken");
+const dotenv = require("dotenv");
+const db = require("../../db/db");
+const tokenService = require("../services/token.service");
+const e = require("express");
```
Comment (usatovaDasha) [inline src/modules/controllers/authorization.controller.js:5]: why need this line?

File: src/modules/controllers/authorization.controller.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const Person = db.person;
+
+dotenv.config();
+
+module.exports.authorization = (req, res, next) => {
+  const accessToken = req.headers.accesstoken;
+  const refreshToken = req.headers.refreshtoken;
+
+  if (accessToken == null) return res.status(422).send("Missing access token");
```
Comment (usatovaDasha) [inline src/modules/controllers/authorization.controller.js:14]: if (! accessToken)....
if (!refreshToken)....

File: src/modules/controllers/authorization.controller.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  jwt.verify(accessToken, process.env.TOKEN_ACC_SECRET, async (err, person) => {
+    if (err) {
+      jwt.verify(
+        refreshToken,
+        process.env.TOKEN_REF_SECRET,
+        async (err, person) => {
+          if (err) return res.status(401).send("Authentication required");
+          const user = await Person.findOne({ where: { id: person.id } });
+          if (refreshToken === user.dataValues.refreshToken) {
+            const { access, refresh } = tokenService.generateToken({
```
Comment (usatovaDasha) [inline src/modules/controllers/authorization.controller.js:27]: why need this destructurization?

File: src/modules/controllers/authorization.controller.js (hunk 1 above)
Comment (DoctorFreed) [reply src/modules/controllers/authorization.controller.js:5]: accidentally added

**Pull request merged**